app/forms/sufia/forms/permission_template_form.rb

```diff
@@ -16,14 +16,6 @@ def visibility_options
          [Hydra::AccessControls::AccessRight::VISIBILITY_TEXT_VALUE_PRIVATE, I18n.t('.restricted', scope: i18n_prefix)]]
       end
 
```

Comment:
This is something @mjgiarlo added, which was required until you did the other work in this PR.